main.cpp

```diff
@@ -421,9 +421,9 @@ HashInfo g_hashes[] =
   { pearson256_test,     256, 0x7F8BEB21, "pearsonhash256",   "Pearson hash, 256-bit SSSE3, low 64-bit", POOR, {}},
 #endif
 #ifdef HAVE_INT64
-  { pearsonb64_test,      64, 0x715717DE, "pearsonbhash64",  "Pearson block hash, 64-bit", POOR, {}},
-  { pearsonb128_test,    128, 0x824DA7ED, "pearsonbhash128", "Pearson block hash, 128-bit, low 64-bit", POOR, {}},
-  { pearsonb256_test,    256, 0xE42B44CE, "pearsonbhash256", "Pearson block hash, 256-bit, low 64-bit", POOR, {}},
+  { pearsonb64_test,      64, 0xB6FF2DFC, "pearsonbhash64",  "Pearson block hash, 64-bit", POOR, {}},
+  { pearsonb128_test,    128, 0x6BEFE6EA, "pearsonbhash128", "Pearson block hash, 128-bit, low 64-bit", POOR, {}},
+  { pearsonb256_test,    256, 0x999B3C19, "pearsonbhash256", "Pearson block hash, 256-bit, low 64-bit", POOR, {}},
```

Comment:
I did not change the rating as I was not sure if only quality drives the rating.

Reply:
Only quality drives the rating. If all tests are fixed it becomes good. Do you want to fix it by yourself?

Reply:
I would prefer to have it evaluated by someone else :wink: